pynucastro/rates/rate.py

```diff
@@ -1854,7 +1854,7 @@ def function_string_cxx(self, dtype="double", specifiers="inline", leave_open=Fa
 
             fstring += "\n"
             for nuc in set(self.rate.reactants + self.rate.products):
-                fstring += f"    Real {nuc}_pf, d{nuc}_pf_dT;\n"
+                fstring += f"    amrex::Real {nuc}_pf, d{nuc}_pf_dT;\n"
```

Comment:
We might as well update this function to use `{dtype}` while we're here

Reply:
indeed.  updated.